packages/bcp/src/cosmwasmconnection.ts

```diff
@@ -324,7 +343,35 @@ export class CosmWasmConnection implements BlockchainConnection {
     } else if (height) {
       txs = await this.cosmWasmClient.searchTx({ height: height }, filter);
     } else if (sentFromOrTo) {
-      txs = await this.cosmWasmClient.searchTx({ sentFromOrTo: sentFromOrTo }, filter);
+      const pendingRequests = new Array<Promise<readonly TxsResponse[]>>();
+      pendingRequests.push(this.cosmWasmClient.searchTx({ sentFromOrTo: sentFromOrTo }, filter));
+      for (const contract of this.erc20Tokens.map(token => token.contractAddress)) {
+        const searchBySender = [
```

Comment:
I like this generic interface for event/tag queries.

The query looks very much like the result, very symetrical and easy to understand